optimum/intel/ipex/modeling_base.py

```diff
@@ -240,129 +145,30 @@ def _from_pretrained(
-            file_name (`str`, *optional*):
-                The file name of the model to load. Overwrites the default file name and allows one to load the model
-                with a different name.
         """
-        if use_auth_token is not None:
-            warnings.warn(
-                "The `use_auth_token` argument is deprecated and will be removed in v5 of Transformers. Please use `token` instead.",
-                FutureWarning,
-            )
-            if token is not None:
-                raise ValueError(
-                    "Both the arguments `use_auth_token` and `token` were specified, which is not supported. Please specify only `token`."
-                )
-            token = use_auth_token
-
-        commit_hash = kwargs.pop("_commit_hash", None)
-
-        model_kwargs = {
-            "revision": revision,
-            "token": token,
-            "cache_dir": cache_dir,
-            "subfolder": subfolder,
-            "local_files_only": local_files_only,
-            "force_download": force_download,
-        }
-
-        if not getattr(config, "torchscript", False):
-            logger.warning("Detect torchscript is false. Convert to torchscript model!")
-
-            if is_torch_version("<", "2.1.0"):
-                raise ImportError("`torch>=2.0.0` is needed to trace your model")
-
-            task = cls.export_feature
-            config.torch_dtype = torch_dtype
-            model = TasksManager.get_model_from_task(
-                task,
-                model_id,
-                library_name="transformers",
-                trust_remote_code=trust_remote_code,
-                torch_dtype=torch_dtype,
-                _commit_hash=commit_hash,
-                **model_kwargs,
+        if getattr(config, "torchscript", False):
+            logger.warning(
+                "IPEXModel will not support torch script model in the future, fallback to TSModelForCausalLM"
             )
+            return TSModelForCausalLM.from_pretrained(model_id, **kwargs)
 
-            return cls(model, config=config, export=True, **kwargs)
-
-        # Load the model from local directory
-        if os.path.isdir(model_id):
-            model_cache_path = os.path.join(model_id, file_name)
-            model_save_dir = model_id
-        # Download the model from the hub
-        else:
-            model_cache_path = hf_hub_download(repo_id=model_id, filename=file_name, **model_kwargs)
-            model_save_dir = Path(model_cache_path).parent
-
-        model = torch.jit.load(model_cache_path)
-        torch.jit.freeze(model.eval())
-
-        return cls(model, config=config, model_save_dir=model_save_dir, **kwargs)
+        model = cls.auto_model_class.from_pretrained(model_id, **kwargs)
+        return cls(model, config=model.config, export=True, **kwargs)
```

Comment:
why would export be needed ?  

```suggestion
        return cls(model, config=model.config, **kwargs)
```

Reply:
done.